xcell/mappers/mapper_KV450.py

```diff
@@ -174,25 +173,16 @@ def get_ellip_maps_mod():
     def get_mask(self, mode=None):
```

Comment:
Shouldn't it be `_get_mask`? (with the underscore)

Reply:
you're right!